test/views/pages/paroleparom1report/ParoleParom1PopupReportPage.java

```diff
@@ -75,7 +75,7 @@ private String fieldNameFromLabelWithLegend(String label, String legend) {
     }
 
     private FluentWebElement findSectionFromLegend(String legend) {
-        return $(xpath(String.format("//fieldset[legend[contains(.,'%s')]]", legend))).first();
+        return $(xpath(String.format("//fieldset[legend[contains(.,'%s')]]", legend))).last();
```

Comment:
Cheeky fix to allow `My title` and `This is My title` so assumes we always want the last one, ie. `My title` will appear under `This is My title` - problem being we have to use `contains`